composer.json

```diff
@@ -20,23 +20,23 @@
         }
     ],
     "require": {
-        "php": "^7.1.3",
-        "symfony/framework-bundle": "^3.4 || ^4.2 || ^5.0",
+        "php": "^7.3 || ^8.0",
+        "symfony/framework-bundle": "^3.4.23 || ^4.2.4 || ^5.0",
```

Comment:
Can be reverted IMHO.

Reply:
Older versions of Symfony had a KernelTestCase which was incompatible with PHPUnit 8. This broke the Travis job which runs with `--prefer-lowest`. See https://travis-ci.org/github/snc/SncRedisBundle/jobs/746572032, https://github.com/snc/SncRedisBundle/pull/599/commits/ae42ace2caee86f0ad251852ac835e8d37de2d68 and https://github.com/symfony/symfony/commit/1077df60ab35c418f8ca196db40500f5489990cd.

Reply:
Makes sense, thanks.